azure-iot-hub/azure/iot/hub/auth.py

```diff
@@ -7,12 +7,15 @@
 """Provides authentication classes for use with the msrest library
 """
 
-from msrest.authentication import Authentication
+from msrest.authentication import Authentication, BasicTokenAuthentication
 from .connection_string import ConnectionString
 from .connection_string import HOST_NAME, SHARED_ACCESS_KEY_NAME, SHARED_ACCESS_KEY
 from .sastoken import SasToken
+from azure.core.pipeline.policies import BearerTokenCredentialPolicy
```

Comment:
You'll need to add the correct azure.core library to setup.py.  This is why your test jobs are failing.

Reply:
Added.